core/src/main/java/com/coderanch/blackjack/Cards.java

```diff
@@ -61,4 +64,22 @@ private Cards() {}
     static Set<Card> getStandardDeck() {
         return STANDARD_DECK;
     }
+
+    static Stack<Card> getShuffledStandardDeck(){
+        return getStandardDeck().stream()
+                .map(c -> new CardPair(RANDOM.nextInt(10_000_000), c))
```

Comment:
Why the range here is `0` .. `10_000_000` (exclusive)? In particular why `10_000_000`?

Reply:
It's arbitrary.  I didn't want duplicate numbers, so I picked a high number to lessen that outcome.

Reply:
The lesson here is: Never use arbitrary values. If you don't have a good reason to use a specific value, either change your design or make the value configurable.

In this case the issue was resolved by a design change.